### PR TITLE
Button

### PR DESCRIPTION
**Pull request by Tanay-J**
Added Buttons:
- Solid buttons
- Outline buttons
- Floating buttons
- Links

image

### CODE REVIEW

**Review by Radheshyam1234 — COMMENTED** [review]
Everything looks good

**Review by kedark152 — APPROVED** [review]
Looks all Good!!

File: stylesheets/base/reset.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+h4,
+h5,
+h6 {
+	font-family: "Poppins", sans-serif;
+	margin: 0;
+	padding: 0.8rem 0;
+}
+
+img {
+	max-width: 100%;
```
Comment (hinanshisuthar) [inline stylesheets/base/reset.css:32]: You can add height: auto as it is a responsive rule to be able to automatically adjust according to width

File: stylesheets/utilities/typography.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
+
+
+.h1, .h2, .h3, .h4, .h5, .h6 {
+    line-height: 1.5;
+    font-weight: bold;
+}
+
+.h1,.text-xxl{
+    font-size:3rem;
+    margin:0.4rem 0;
```
Comment (hinanshisuthar) [inline stylesheets/utilities/typography.css]: use px instead of responsive units for values <=10

File: components/button.css [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+
+.btn:hover {
+    filter: brightness(95%);
+    box-shadow: rgba(0, 0, 0, 0.1) 0px 4px 6px -1px, rgba(0, 0, 0, 0.06) 0px 2px 4px -1px;
+}
+.btn-float:hover{
+    filter: brightness(95%);
+    box-shadow: rgba(0, 0, 0, 0.1) 0px 4px 6px -1px, rgba(0, 0, 0, 0.06) 0px 2px 4px -1px;
+}
```
Comment (hinanshisuthar) [inline components/button.css]: add a new line at the end of file

**Review by hinanshisuthar — CHANGES_REQUESTED** [review]
Add a new line at the end of file

**Pull request merged**